src/views/Gallery.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div id="gallery-content">
+        <div id="header-mobile">
+            <div>
+                <h2>Specimen gallery</h2>
+                <h3>Self similarity telescope</h3>
+            </div>
+            <div type="button" id="change-button">
+                <span
+                    alt="Swap icon"
+                    id="change-icon"
+                    class="material-icons-sharp"
+                    >swap_horiz</span
+                >
+                <p id="change-text">Change Visualizer</p>
+            </div>
```

Comment:
What is the design purpose of the change button? It's not hooked up to anything yet, correct? Would it be more conservative to leave it out of the view until it is ready to be hooked up to some functionality (especially if the functionality is still somewhat design-in-progress)? On the other hand, if it has a clearly defined and pretty well finalized intended function, fine to leave it in as a placeholder, but perhaps comment what that functionality will be. Thanks.

Reply:
The functionality for this is indeed finalized so its currently just a placeholder but it will soon have functionality and we saw no reason to delay its visual implementation.

Reply:
Sorry if I did not make my question clear: Why do these preview "cards" for specimens have a "Change Visualizer" button? I understand that one may want to change the visualizer when in the specimen editor, but why does the design call for directly changing the visualizer of a particular specimen from the Gallery? To me, the Gallery seems like the room in the museum where the "finished" artworks are hung, so I wouldn't expect to see ways to change them there. When I put one artwork up on my "easel" -- the specimen editor -- then I'd expect to see the ability to edit it. Or am I misunderstanding where the Change Visualizer button(s) are or what they do? Are they simply the "Edit" button for each specimen that brings it up in the Specimen Editor? If so, then I'd suggest that a label like "Edit" would be clearer than "Change Visualizer", which to me implies that I am going to take that specimen in-place in the Gallery and choose a different visualizer for it on the spot. Hopefully I've expressed myself more clearly now. Thanks!

Reply:
Ah now I understand the question. So the change visualizer button is only intended for mobile since the mobile UI will only display specimens for one given visualizer at a time. It won't be scrolling based like in the desktop version. So the change visualizer button essentially just allows users to select what visualizer the specimens they are seeing (be they featured or saved) are using. It has nothing to do with editing the specimens individually its simply a sorting criterion. I hope that makes sense. I will raise it with the team if we want to potentially rename this button as I also found it confusing at first.

Reply:
OK, maybe "Select Visualizer"? Or "Gallery Rooms"/"Other Rooms"? The latter is agnostic as to the exact sorting criterion, which would be helpful if different criteria are ever allowed. With this clarification, though, I will proceed with the review; however, feel free to change it at any point, prior to this PR being merged or in a small one-off PR afterward. If this goes into "main" with the potentially confusing name, please file an issue that it should be revisited.

Reply:
We have decided to change the text to say "Select Visualizer". Thank you for the suggestion. This change has been implemented in my latest commit.